### ISSUE TITLE
help link for ai tester

### ISSUE THREAD

**Issue by john-honovich** [ipvideomarketinfo/ai-model-tester#25]
somewhere near the bottom, we should have some link to either a forum for asking questions and/or contact form to ask questions

not sure placeholder for later
<img width="673" alt="screen shot 2019-02-13 at 6 46 07 pm" src="https://user-images.githubusercontent.com/8518367/52754322-c0b72d80-2fbf-11e9-99f4-2cd5d007fb4b.png">

**Comment by lefnire** [comment]
create a new forum thread / entire forum? Or link them to something existing? 

I can create a webform contacting me if that's preferred

**Comment by john-honovich** [comment]
nah, we'll just create a discussion in the main forum and link to that

we already have the contact webform set up so it can be sent custom messages (e.g., help request for AI tester" etc. and we'll want @ryanipvm to do 1st level support forwarding to us as needed for more advanced requests
